.thoth.yaml

```diff
@@ -7,7 +7,7 @@ runtime_environments:
     operating_system:
       name: ubi
       version: "8"
-    python_version: "3.6"
+    python_version: "3.8"
```

Comment:
@fridex @harshad16 do I need to do this change in `.thoth.yaml` for all repositories as well?

Reply:
Yes, please do so. :+1:

Reply:
thanks